th2_common/schema/factory/common_factory.py

```diff
@@ -152,14 +151,56 @@ def create_from_kubernetes(namespace, box_name):
         dictionary_path = config_dir + '/' + 'dictionary.json'
         prometheus_path = config_dir + '/' + 'prometheus.json'
 
+        prometheus_host = '0.0.0.0'
+        prometheus_port = 9752
+
+        try:
+            mkdir(config_dir)
+            logger.info('Directory %s is created at %s' % (config_dir, getcwd()))
+        except OSError:
+            logger.info('All configuration in the %s folder are overridden' % (getcwd() + '/' + config_dir))
+
+        CommonFactory._get_grpc(config_maps_dict, box_name, grpc_path, services_dict, kube_contexts)
+        CommonFactory._get_custom(config_maps_dict, box_name, custom_path)
+        CommonFactory._get_mq(config_maps_dict, box_name, mq_path)
+        CommonFactory._get_cradle(config_maps_dict, cradle_path, kube_contexts, service_in_namespace_service)
+        CommonFactory._get_rabbit(config_maps_dict, rabbit_path, kube_contexts, service_in_namespace_service)
+        CommonFactory._get_dictionary(box_name, v1.list_config_map_for_all_namespaces(), dictionary_path)
+
+        with open(prometheus_path, 'w') as prometheus_file:
+            prometheus_file.write('{' + f'"host":"{prometheus_host}","port":{prometheus_port},"enabled":true' + '}')
```

Comment:
Not sure if we need this and default values above, because they're described in PrometheusConfiguration class